python/pythonizations/SmartPtr.C

```diff
@@ -1,18 +1,18 @@
 #include "SmartPtr.h"
 
-auto_ptr<MyShareable> gime_mine() { return mine; }
-auto_ptr<MyShareable>* gime_mine_ptr() { return &mine; }
-auto_ptr<MyShareable>& gime_mine_ref() { return mine; }
+shared_ptr<MyShareable> gime_mine() { return mine; }
+shared_ptr<MyShareable>* gime_mine_ptr() { return &mine; }
```

Comment:
good question: the warnings appear in all builds (e.g. http://cdash.cern.ch/testDetails.php?test=19878020&build=360727), arm actually crashes:

```
CMake Error at /data/sftnight/build/workspace/root-nightly-master/BUILDTYPE/Optimized/COMPILER/native/LABEL/arm64/root/cmake/modules/RootTestDriver.cmake:161 (message):
  error code: Child aborted
```

My commit should (but I can't test because the jenkins bot does not run on roottest PRs) resolve the warnings, which might or might not help with the failure on arm. Anyway, as I said in my first message feel free to close this if you don't agree on the approach.